app/build.gradle

```diff
@@ -141,8 +141,8 @@ dependencies {
       'androidx.test.ext:junit:1.1.1',
       'com.github.bumptech.glide:mocks:4.11.0',
       'com.google.truth:truth:0.43',
-      'org.robolectric:annotations:4.3',
-      'org.robolectric:robolectric:4.3',
+      'org.robolectric:annotations:4.4',
+      'org.robolectric:robolectric:4.4',
```

Comment:
We should update to 4.4 in all modules in one shot.

Reply:
Done